node/bft/ledger-service/src/ledger.rs

```diff
@@ -343,9 +345,49 @@ impl<N: Network, C: ConsensusStorage<N>> LedgerService<N> for CoreLedgerService<
             metrics::gauge(metrics::bft::LAST_COMMITTED_ROUND, block.round() as f64);
             metrics::increment_gauge(metrics::blocks::SOLUTIONS, num_sol as f64);
             metrics::increment_gauge(metrics::blocks::TRANSACTIONS, num_tx as f64);
+            self.update_block_metrics(block);
         }
 
         tracing::info!("\n\nAdvanced to block {} at round {} - {}\n", block.height(), block.round(), block.hash());
         Ok(())
     }
+
+    #[cfg(feature = "metrics")]
+    fn update_block_metrics(&self, block: &Block<N>) {
```

Comment:
To track Aborted Transactions & Aborted Solutions. Block has two methods:

`aborted_solution_ids` and `aborted_transaction_ids` - you can use these two methods to count the aborted solutions & transactoins.

Reply:
I used those in the function already, they are the last two lines

Reply:
Nice! Let me take a second look.